thorlcr/thorutil/thmem.cpp

```diff
@@ -1619,6 +1619,7 @@ class CThorRowCollectorBase : public CSpillable
 protected:
     CThorSpillableRowArray spillableRows;
     IPointerArrayOf<CFileOwner> spillFiles;
+    Linked<CFileSizeTracker> tempFileSizeTracker;
```

Comment:
minor: Don't really need to save because it can be accessed via activity member.